internal/pipe/publish/publish.go

```diff
@@ -49,6 +50,7 @@ var publishers = []Publisher{
 	krew.Pipe{},
 	scoop.Pipe{},
 	milestone.Pipe{},
+	chocolatey.Pipe{},
```

Comment:
should probably be closer to scoop, as the milestone is documented to be the last part if I'm not mistaken